app/views/shared/_splash.html.erb

```diff
@@ -0,0 +1,13 @@
+<section class="d-flex flex-column align-items-center justify-content-center" style="height:100vh;">
+  <div>
+    <h1>Budget App</h1>
+  </div>
+  <div class="d-flex flex-column gap-2 mt-5">
+    <div>
+      <%= link_to "LOG IN", new_user_session_path, class:"btn btn-primary w-100" %>
+    </div>
+    <div>
+      <%= link_to "SIGN UP", new_user_registration_path, class:"btn btn-primary w-100" %>
+    </div>
```

Comment:
- [ ] Great job creating a splash page, unfortunately, your design does not match the given design. I have attached images for your reference.

## Yours

![yours](https://github.com/enis-memic/budget-app/assets/53124548/5a69533f-58dd-4b86-b64c-42afdc1350d1)

## design

![og](https://github.com/enis-memic/budget-app/assets/53124548/b1d7906e-4eb9-4215-9f56-c33cb1285ad4)

Reply:
@elisha2kyakpo1 Thank you for these suggestions!